index.html

```diff
@@ -301,111 +301,78 @@
-  </p>
-
-    <section> <h3>Chain of Trust</h3>
-    <p>
-      Things discoverable and accessible in a WoT network SHOULD be identified and authenticated.
-    </p>
-    <p>
-      The integrity of WoT communication SHOULD be ensured by implementations.
-    </p>
-    </section>
-
-    <section> <h3>Threat Model</h3>
-    <p>
-    The main threats are summarized in the [Security and Privacy document](http://w3c.github.io/web-nfc/security-privacy.html#threats-and-possible-solutions).
-    </p>
-    <p>
-      In this specification the following threats are considered of the highest priority:
-      <ul>
-        <li>
-          Unauthorized gathering of user-sensitive information.
-        </li>
-        <li>
-          Vulnerability of the underlying protocols.
-        </li>
-        <li>
-          Protect the integrity of <a>Thing</a>s and <a>Thing Description</a>s.
-        </li>
-      </ul>
-    </p>
-    </section>
-
-    <section> <h3>Security Mechanisms</h3>
-      <section> <h4>Identification, Authentication, Authorization</h4>
-      </section>
-
-      <section> <h4>Transport-level security</h4>
-      </section>
-
-      <section> <h4>Application-level security</h4>
-      </section>
-    </section>
-
-    <section class="informative"> <h3>Security policies</h3>
-      <p>
-        This section summarizes the security policies which are specified as
-        normative requirements in the respective algorithms of this
-        specification.
-      </p>
-    </section>
-  </section>
-
   <section> <h2>WoT Data Representation</h2>
   <p>
   </p>
   </section>
 
   <section> <h2>The WoT API</h2>
   <p>
-    The API object provides access obtain <a>Thing</a>s by discovery or creation. A <a>Thing</a> is represented either as a <a>ConsumedThing</a> or <a>ExposedThing</a>.
+    The API object represents an implementation of the <a>WoT Runtime</a> and provides functionality to obtain <a>Thing</a>s by discovery or creation.
   </p>
   <p>
     The <dfn>WoT object</dfn> exposes only functions and has no internal state.
   </p>
-  <p class="issue">
-     Because of this, implementations MAY use a global object, or provide a constructor, or provide the API object, or by other means (e.g. by `require()`). Since the common denominator across various <a>Runtime</a> types would be to use a constructor, this would be recommended. As a consequence, the <a>ConsumedThing</a> and <a>ExposedThing</a> objects could also be constructed, complete with an asynchronous fetch functionality.
-  </p>
   <p class="note">
-    The WebIDL interfaces presented in this initial version of the document are based on the [[!WOT-PRACTICES]]. It is expected to be updated soon.
+     Browser implementations SHOULD use a namespace object such as `wot`, and [Node.js](https://nodejs.org/en/)-like runtimes MAY provide the API object through the [`require()`](https://nodejs.org/api/modules.html) mechanism.
```

Comment:
See comment below. Rationale:
* can be global namespace for all kind of runtimes
* design decision  for implrmrmnter
* ES2015 ``import`` additionally to ``require()``

Suggestion:
`` Browser implementations SHOULD use a namespace object such as `wot`, and [Node.js](https://nodejs.org/en/)-like runtimes MAY alternatively provide the API object through the [`require()`] (https://nodejs.org/api/modules.html)  or [`import`](http://www.ecma-international.org/ecma-262/6.0/#sec-imports) mechanism.``